### ISSUE TITLE
Naming conflict with fdrtd client library 

### ISSUE THREAD

**Issue by UNakade** [fdrtd/fdrtd#4]
If the fdrtd client library happens to be installed in the environment from which `python -m fdrtd.webserver` is called, Python tries to find a webserver module in the fdrtd client library, and the command fails. The implicit namespace package seems to be getting a lower priority than an installed library. Adding `__init__.py` to the fdrtd directory seems to fix the problem, but having this naming conflict might create issues down the line.

**Comment by Ballhausen** [comment]
Yes, this is the case. We deliberately changed the name of the main repo from frdtd/server to fdrtd/fdrtd to highlight it. Server-side, imports now are much more meaningful: `import fdrtd.client.api`, `import fdrtd.server.discovery` and so on.

The drawback, as you correctly state, is a naming collition with the Pypi package fdrtd which holds the client library.
There are several solutions:

1) There is no need to have the fdrtd package installed in the server environment in the first place. And it probably should not be installed globally, either. So pip uninstall it and only install it in virtual environments for client scripts (if they even reside on the same logical or physical machine). Not having the fdrtd package installed is also the natural state for a new user, so no problem here. We would add a remark to the installation guide that the server code and client package do not mix well.
2) Rename the fdrtd package to something different like "fdrtd_client", "fdrtd_api", or "federated".
3) Rename the main repo to something different like "fdrtd_server".
4) Adding `__init__.py` seems to be a machine dependent fix/hack, and should not be the solution.
5) ...?

My preffered solution by far is 1), but I am open to opinions here.

**Comment by UNakade** [comment]
I think the most important code we have will be on the server side, so it makes sense to name it fdrtd. I also agree that this makes imports on the server side more meaningful. But it seems a little weird to me that two separate libraries with two different functions are named the same, especially because they are so interdependent. Someone might want to test these out before deploying a server. It could take time to debug this error if there is an older leftover installation of the fdrtd client. Having to set up a different virtual environment just to see how this works sounds like an unnecessary hurdle. Also, while the client side library is named fdrtd in Pypi, we are using a different name for the same code here, which is stored in the "client" folder of this repo. At least in my opinion, we should have different names for the two libraries.

**Comment by Ballhausen** [comment]
Okay, we have a resolution here.
The client library will not be named fdrtd anymore. In fact, we will switch alltogether to a more general client-server framework to drive the server-side microservices.
These changes will be coming with 0.3.0 and I will update this issue as the changes go live.
